src/main/webapp/resources/js/pages/launch/ParameterWithOptions.jsx

```diff
@@ -15,7 +16,9 @@ import { Checkbox, Form, Radio, Select } from "antd";
  * @returns {*}
  * @constructor
  */
-export function ParameterWithOptions({ parameters }) {
+export function ParameterWithOptions() {
+  const [{ parameterWithOptions: parameters }] = useLaunch();
```

Comment:
Another question. What is this? I haven't seen this before (the format)

Reply:
This is [destructing with renaming](https://wesbos.com/destructuring-renaming).  Essentially shorthand for:

```
const [state] = useLaunch(); // Array destructure;
const {parameterWithOptions : parameters} = state; // Object destructure with renaming to paramters
```

Reply:
Awesome. Thanks!